.travis.yml

```diff
@@ -27,6 +27,7 @@ env:
   global:
   - PATH=$PATH:$HOME/gcloud/google-cloud-sdk/bin
   - GOOGLE_APPLICATION_CREDENTIALS=$TRAVIS_BUILD_DIR/client_secrets.json
+  - GCP_PROJECT_ID=cloud-samples-tests
```

Comment:
Should this use `GCLOUD_PROJECT`?  All gcloud-* libraries check `GCLOUD_PROJECT` for project name (if not specified explicitly in code) so it's a conventional environment variable for this.